data/templates/default/init.lua

```diff
@@ -780,5 +797,234 @@ return {
+         space_after = ingame_heading_space_after,
+      },
+      ingame_heading_3 = {
+         font = {
+            color = ingame_heading_color,
+            face = ingame_font_face,
+            size = 13,
+         },
+         space_before = ingame_heading_space_before_small,
+         space_after = ingame_heading_space_after,
+      },
+      ingame_heading_4 = {
+         font = {
+            color = ingame_heading_color,
+            face = ingame_font_face,
+            size = 12,
+            italic = true,
+         },
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after,
+      },
+      ingame_text = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after,
+      },
+      ingame_sparse = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after,
+      },
+      ingame_image_line = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after + ingame_padding,
+      },
+      ingame_lore_author = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 10,
+            italic = true,
+         },
+         align = "right",
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after + 3,
+      },
+      about_text = {
+         font = {
+            color = fs_font_color,
+            face = fs_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after,
+      },
+      about_sparse = {
+         font = {
+            color = fs_font_color,
+            face = fs_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after,
+      },
+
+
+      -- Returned when lookup by name fails.
+      -- Make this stand out to help debugging
+      unknown = {
+         font = {
+            color = unknown_font_color,
+            face = unknown_font_face,
+            size = unknown_font_size,
+            italic = true,
+         },
+         space_before = unknown_font_size,
+         space_after = unknown_font_size,
+      },
    },
+
+   colors = {
+      campaign_bar_thron = campaign_leader_color,
+      campaign_bar_boldreth = {0xFF, 0x00, 0x00},
+      campaign_bar_khantrukh = {0x80, 0x80, 0xFF},
+
```

Comment:
Can you please use decimal values here? (and also below)

So each color definition has the same format and the values are not mixed up.

Reply:
Do we need decimal, or can hexadecimal be the unified format? These are hexadecimal in the current campaign scripts (which is needed for richtext formatting), so reviewing should be easier this way.

Reply:
In my opinion the lua files are the part of the game where people without much knowledge of programming can work with. So they should be as easy as possible for them imho.

Reply:
OK, I will convert all to decimal then.